index.html

```diff
@@ -17,23 +17,27 @@
         <li class="menu__item" data-js="dropdown-about">
           <a href="#">ABOUT</a>
         </li>
-        <li class="menu__item menu__item--aboutmenu--hidden" data-js="about-menu">
-          <a href="#">HISTORY</a>
-        </li>
-        <li class="menu__item menu__item--aboutmenu--hidden" data-js="about-menu">
-          <a href="#">OUR TEAM</a>
-        </li>
-        <li class="menu__item menu__item--aboutmenu--hidden" data-js="about-menu">
-          <a href="#">LOCATIONS</a>
-        </li>
+        <!-- Would prefer to use another list element here -->
+        <!-- Howerver this would involve quite a bit more CSS -->
+        <div class="menu__item--aboutmenu menu__item--aboutmenu--hidden" data-js="about-menu">
```

Comment:
Establishing a hierarchy by adding a container element for all of the elements that need to be conditionally shown/hidden.